src/nimblepkg/nimscriptsupport.nim

```diff
@@ -382,7 +382,7 @@ proc execScript(scriptName: string, flags: Flags, options: Options): PSym =
 
   # Ensure that "nimblepkg/nimscriptapi" is in the PATH.
   block:
-    let t = getTempDir() / "nimblecache"
+    let t = getNimbleTempDir() / "nimblecache"
```

Comment:
This defeats the purpose of the cache, `getNimbleTempDir` will return a different directory for each Nimble invokation...

Reply:
Oh! Duh. Ok.

Reply:
How about now?